tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
I've done that on purpose - currently we have 2 tests, which are trying to install some package manually, but future content host tests, for example, tests from issues #1709 and #1710, will verify automatic package installation/upgrade/deletion. That means that I need VM without some specific package installed for each of these tests to avoid conflicts.